Extensions/Guardrail+Settings.swift

```diff
@@ -23,7 +23,7 @@ public extension Guardrail where Value == HKQuantity {
         .min()!
     }
 
-    static let correctionRange = Guardrail(absoluteBounds: 87...180, recommendedBounds: 101...115, unit: .milligramsPerDeciliter)
+    static let correctionRange = Guardrail(absoluteBounds: 87...180, recommendedBounds: 101...115, unit: .milligramsPerDeciliter, startingSuggestion: 100)
```

Comment:
Why is the starting suggestion outside the recommended bounds?

Reply:
Two things: 1) The Tidepool bounds are being updated to start at 100, instead of 101. 2) The starting suggestion is really only used in DIY, since Tidepool users will be starting with their provider's recommendation. 

Also, Guardrails are something that are likely to diverge at some point, so DIY will be able to have different settings than us.

Reply:
Rick was already tackling that, and I think it's already landed, and will be merged in before this lands.